tools/lsp/fmt/fmt.rs

```diff
@@ -1407,6 +1438,27 @@ component ABC {
         "fifth string"
     ];
 }
+"#,
+        );
+    }
+
+    #[test]
+    fn property_animation() {
+        assert_formatting(
+            r#"
+export component MainWindow inherits Window {
+    animate background { duration: 800ms;}
+    Rectangle {}
+}
+"#,
+            r#"
+export component MainWindow inherits Window {
+    animate background {
+        duration: 800ms;
+    }
```

Comment:
In the next pr, I have changed it so that if there is only binding it will be kept on a single line, but with multiple it is broken into multiple lines.

But I can change it to 80 char limit too if that's better.